frontend/src/export/mtga.spec.js

```diff
@@ -0,0 +1,38 @@
+const {describe, it} = require("mocha");
+const assert = require("assert").strict;
+
+import { exampleDeck as deck } from "./test-cards.js";
+import mtga from "./mtga.js";
+
+describe("export/mtga", () => {
+  it("download of *** file", () => {
+    assert.equal(mtga.download, undefined, "download disabled");
+  //   assert.equal(
+  //     mtga.download("My deck", deck),
+  //     downloadOutput(),
+  //     "correct output"
+  //   );
```

Comment:
Why is this commented? Should it be deleted ?

Reply:
removed